resources-compose/src/jvmMain/kotlin/dev/icerock/moko/resources/compose/StringDescExt.kt

```diff
@@ -4,9 +4,11 @@
 
 package dev.icerock.moko.resources.compose
 
+import androidx.compose.runtime.Composable
 import dev.icerock.moko.resources.desc.StringDesc
 
 @Suppress("EXTENSION_SHADOWED_BY_MEMBER")
+@Composable
```

Comment:
why?

Reply:
Don't know, @KuzminVik updates for kotlin 1.9.0, removed
Viktor, can u help here?

Reply:
Oh, wait a sec, i'm find need because without this annotation has crash with exception:
e: file:///Users/exndy/Documents/Projects/moko-resources/resources-compose/src/jvmMain/kotlin/dev/icerock/moko/resources/compose/StringDescExt.kt:11:23 Mismatched @Composable annotation between expect and actual declaration

Reply:
@Lobynya attention